src/physics/physics.cpp

```diff
@@ -653,16 +654,55 @@ void initPhysics() {
 	trackShape->setSimulationFilterData(trackSimFilterData);
 
 	PxRigidStatic* track = gPhysics->createRigidStatic(trackShape->getLocalPose());
+	track->setName("Track");
 	track->attachShape(*trackShape);
 	gScene->addActor(*track);
 
 	// Create vehicle that will drive on the plane. (This one is the player)
 	initVehicle();
 }
 
+int caster = 0;
+
 void stepPhysics() {
 	const PxF32 timestep = 1.0f / 60.0f;
 
+	if (caster++ == 60) {
+		// do raycast
+
+		PxU32 maxHits = 1;
```

Comment:
maxHits and hitFlags are never used, they were from some previous experimentation on a different ray cast method I was testing, feel free to remove them